python-qa-tools/load-test/multi_msg_load.py

```diff
@@ -0,0 +1,103 @@
+import argparse, os, sys, time
+from modules.bank.query import query_balances
+from utils.bank import print_balance_deductions
+from utils.commands import exec_command
+from utils.keys import fetch_account_address
+from utils.txs import fetch_seq_no, signed_tx, unsigned_tx, write_json
+from utils.types import account_type, num_txs_type
+
+DAEMON = os.getenv('DAEMON')
+DENOM = os.getenv('DENOM')
+CHAINID = os.getenv('CHAINID')
+DAEMON_HOME = os.getenv('DAEMON_HOME')
+HOME = os.getenv('HOME')
+
+parser = argparse.ArgumentParser(description='This program takes inputs for intializing multi message load test.')
+parser.add_argument('FROM', type= account_type, help= 'From which account the transaction should be intialized')
+parser.add_argument('TO', type= account_type, help= 'Reciever account number.')
+parser.add_argument('NUM_TXS', type = num_txs_type, help= 'Number of transactions to be made, Min. should be 1000')
+
+args = parser.parse_args()
+FROM, TO, NUM_TXS = int(args.FROM), int(args.TO), int(args.NUM_TXS)
+
+if FROM == TO:
+    sys.exit('Error: The values of arguments "TO" and "FROM" are equal make sure to set different values')
+
+RPC, num_msgs = "http://127.0.0.1:16657", 30
+
+#### Fetching Bech32 addresses ######
+
+acc1, acc1err = fetch_account_address(f"account{FROM}")
+if len(acc1err):
+    sys.exit(acc1err)
+
+acc2, acc2err = fetch_account_address(f"account{TO}")
+if len(acc2err):
+    sys.exit(acc2err)
+
+#### Fetch Balances of acc1 acc2 before execting the load test ####
+before_acc1_balance, before_acc1_balanceerr = query_balances(acc1, RPC, amount = True)
+if len(before_acc1_balanceerr):
+    sys.exit(before_acc1_balanceerr)
+
+before_acc2_balance, before_acc2_balanceerr = query_balances(acc2, RPC, amount = True)
+if len(before_acc2_balanceerr):
+    sys.exit(before_acc2_balanceerr)
+
+#### Fetching sequence numbers of to and from accounts
+os.chdir(os.path.expanduser(HOME))
+status, seq1no = fetch_seq_no(acc1, RPC)
+if not status:
+    sys.exit(seq1no)
+
+status, seq2no = fetch_seq_no(acc2, RPC)
+if not status:
+    sys.exit(seq2no)
+
+#### Generating unsigned transactions with a single transfer message 
+for i in range(0, int(NUM_TXS)):
+    status, unsignedTxto = unsigned_tx(acc1, acc2, 'unsignedto.json')
+    if not status:
+        print(unsignedTxto)
+    
+    status, unsignedTxfrom = unsigned_tx(acc2, acc1, 'unsignedfrom.json')
+    if not status:
+        print(unsignedTxfrom)
+#### Duplicating and appending transfer message in the existing array to create a multi-msg transaction        
+    for j in range(0, int(num_msgs)):
+        write_json('unsignedto.json')
```

Comment:
Let's have a util function to append messages and write once at the end of the loop
```suggestion
        write_msgs_to_file(filename, msgs)
```